### PR TITLE
Reorganize build/ and dist/

### LINKED ISSUE #399: Reorganize bundle and build
- [x] Packages generated should be in `build` folder, not in `dist`
- [x] JS/CSS/assets/HTML should be all in `dist`
- [x] Use [HtmlWebpackPlugin](https://webpack.js.org/plugins/html-webpack-plugin/) to generate the HTML page, drop `dist/index.html`

### PR DESCRIPTION
**Pull request by KeitIG**
Fix #399

### CODE REVIEW

**Comment by KeitIG** [comment]
mmmmh I do not get this sass-lint error...

**Comment by YurySolovyov** [comment]
`spawn ENOENT` means it does not see `sass-lint` binary/executable

**Comment by KeitIG** [comment]
Debugging the build via SSH give some interesting results. I still get the same error, but when I `rm -rf node_modules && npm install && npm run lint:sass`, everything runs fine.

**Comment by KeitIG** [comment]
Looks like rebuilding without cache solve the issue, let's see how things go in the future.

File: package.json [hunk 1]
```diff
@@ -120,6 +124,7 @@
     "eslint-plugin-react": "7.5.1",
     "extract-text-webpack-plugin": "3.0.2",
     "file-loader": "1.1.6",
+    "html-webpack-plugin": "^2.30.1",
```
Comment (YurySolovyov) [inline package.json]: Exact version

**Pull request merged**